File: src/test/java/br/com/model/classes/NovoPedidoTest.java
package br.com.model.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.classes.NovoPedido;

public class NovoPedidoTest {
    @Test
    void testNovoPedido() {
        System.out.println(NovoPedido.novoPedido().toString());
    }
}

File: src/test/java/br/com/model/NovoPedidoTest.kt
package br.com.model

import br.com.newObjects.model.NovoPedido
import org.junit.jupiter.api.Test

class NovoPedidoTest {
    @Test
    fun testNovoPedido() {
        println(NovoPedido.novoPedido().toString())
    }
}
